File: Relogio.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    int h, m, s; cin >> h >> m >> s;
    int t; cin >> t;

    t += s;
    t += m * 60;
    t += h * 3600;

    int RespSegungos = t % 60;
    t /= 60;
    int RespMinutos = t % 60;
    t /= 60;
    int RespHora = t % 24;

    cout << RespHora << "\n" << RespMinutos << "\n" << RespSegungos << "\n";
}

File: Relogio.py
h = int(input())
m = int(input())
s = int(input())
t = int(input())

t += h * 3600
t += m * 60
t += s

RespSegungos = t % 60
t = t // 60
RespMinutos = t % 60
t = t // 60
RespHora = t % 24

print(RespHora)
print(RespMinutos)
print(RespSegungos)
